integration_test/statictest/tests.py

```diff
@@ -9,7 +9,7 @@
 
 class CollectStaticTest(SimpleTestCase):
     def setUp(self):
-        self.temp_dir = tempfile.TemporaryDirectory()
+        self.temp_dir = tempfile.TemporaryDirectory(prefix='dsc_')
```

Comment:
Why this needs to be changed?

Reply:
a windows box has too many folders in temp. with prefix it is simpler to find the right one